README.md

```diff
@@ -18,26 +18,27 @@
 1. [Scenario specific samples](#Scenario-specific-samples)
 
 
-## Try it yourself - experience the Apps in your Microsoft Teams client
+## Try it yourself - experience the Apps in your Microsoft Teams client.
 
 Here are all the samples which are deployed on Microsoft Azure and you can try it yourself by uploading respective app packages (.zip files links below) to one of your teams and/or as a personal app. (Sideloading must be enabled for your tenant; [see steps here](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading).) These apps are running on the free Azure tier, so it may take a while to load (warm up) if you haven't used it recently and it goes back to sleep quickly if it's not being used. Rest assured, once it's loaded it's pretty snappy.
 
+Note: With sample Name added * Admin consent follow the links (https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/grant-admin-consent?pivots=portal).
```

Comment:
Note: Need admin consent which samples name added with asterisk (*). Please follow the **links** to provide admin consent.
Link should be clickable.

Reply:
resolved